torchmetrics/metric.py

```diff
@@ -30,7 +30,7 @@
 from torchmetrics.utilities.imports import _LIGHTNING_AVAILABLE, _compare_version
 
 
-def distributed_available() -> bool:
+def jit_distributed_available() -> bool:
```

Comment:
jit sounds like something related to TorchScript, but I see what you mean and it makes sense in a literal sense of it.